src/factory.js

```diff
@@ -137,7 +137,13 @@ module.exports = ({ Component, createElement, PropTypes }) => {
 
                   propsSpy.clear()
                   stateSpy.clear()
-                  previousValue = c(stateProxy, propsProxy)
+
+                  try {
+                    previousValue = c(stateProxy, propsProxy)
+                  } catch (error) {
+                    noop()
```

Comment:
What does it do?

Reply:
Nothing. According to my tests, when I wrap `previousValue = c(stateProxy, propsProxy)` with `try and catch` it don't trigger a render and still expose the previous value in subsequent render as it descriped in issue https://github.com/JsCommunity/reaclette/issues/21
So I added `noop()` to say that when we catch an error, we do nothing.

Reply:
Do not call a function if there is nothing to do.

It's better to add a comment explaining why you are swallowing the error.